gradle.properties

```diff
@@ -1,10 +1,13 @@
-org.gradle.jvmargs=-Xmx1G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx3G -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 org.gradle.java.installations.fromEnv=JDK_8
 
 group=org.jetbrains.kotlinx
 version=0.6.0-RC.2
 versionSuffix=SNAPSHOT
 
+timezonesMajorVersion=1.0
+tzdbVersion=2024a
```

Comment:
Note (we'll have to discuss this internally): SemVer doesn't allow letters in the initial components, so `1.0.2024a` is an invalid version number. We could stick to this anyway, or we could use the less natural `1.0.2024-a` (note the dash).

Reply:
In the end, this isn't a SemVer anyway, so dashes aren't needed, `2024a` is ok.